project/src/main/world/career-world.gd

```diff
@@ -47,7 +77,23 @@ func _ready() -> void:
 	_move_camera()
 
 
+# Loads the cutscene's environment, replacing the current one in the scene tree.
+func prepare_environment_resource() -> void:
+	_loaded_environment_path = _career_environment_path()
+	EnvironmentScene = load(_loaded_environment_path)
+
+
+func _career_environment_path() -> String:
+	var environment_name := PlayerData.career.current_region().environment_name
+	return ENVIRONMENT_PATH_BY_NAME.get(environment_name, DEFAULT_ENVIRONMENT_PATH)
+
+
 func refresh_creatures(pickable_career_levels: Array) -> void:
```

Comment:
Rename this method, it is no longer `refresh_creatures`. It also warrants a comment.

Reply:
Renamed to `refresh_from_career_data` and added a comment.